src/praisonai-agents/praisonaiagents/tools/github_tools.py

```
@@ -7,15 +7,15 @@
 @tool
 def github_create_branch(branch_name: str) -> str:
     """Create and checkout a new git branch.
-    
-logger.debug(f"Branch '{branch_name}' checked out successfully.")
+
     Args:
         branch_name: The name of the branch to create and checkout.
     """
     try:
         # Check if we are in a git repository
         subprocess.run(["git", "rev-parse", "--is-inside-work-tree"], check=True, capture_output=True)
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)